workflow_orchestrator.py

```
@@ -0,0 +1,324 @@
+This module orchestrates the complete research workflow:
+1. Sync from Zotero (download tagged PDFs to to-read folder)
+2. Upload to reMarkable (sync to-read folder)
+3. Download from reMarkable (sync annotated PDFs to read folder)
+4. Process annotations (batch OCR and generate markdown/HTML)
+
+This provides a complete end-to-end solution for managing research papers
+from library to annotated output.
+
+"""
+
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)